app/app.view.ts

```diff
@@ -35,10 +35,18 @@ namespace $.$$ {
 			return formula_html( str )
 		}
 
+		@ $mol_mem_key
+		card_link( n: number ) {
+			let link = this.fetch_by_number( n )?.data[ 0 ]?.attributes?._gnome_material_id
+			return `https://optimade-gnome.odbx.science/v1/structures/data/gnome_data/by_id.zip/data/gnome_data/by_id/${link}.CIF`
+		}
+		
+
 		@ $mol_mem_key
 		card_loaded( n: number ) {
 			try {
 				this.card_name( n )
+				this.card_link( n )
```

Comment:
this is not necessary, `card_name` is put into this method just to wait for the fetch to done (it's worth replacing `card_name` with `fetch_by_number` to make this more obvious)